### DISCUSSION TITLE
How to enable Wayland?

### DISCUSSION THREAD

**Discussion by HubKing** [Kong/insomnia#6613] [Help] [answered]
It's blurry under Xwayland. I have passed `--enable-features=UseOzonePlatform --ozone-platform=wayland` but that made no window border under Gnome. What is the recommended way to enable Wayland?

**Comment by musakarimli** [comment, marked as answer]
Can you try with this flag? It worked for me

--enable-features=UseOzonePlatform,WaylandWindowDecorations --ozone-platform=wayland

**Reply by HubKing** [reply]
Thanks.

Is there a way also to enable IME (fcitx, etc) to type Asian languages? `--gtk-version=4` crashes it.

**Reply by washanhanzi** [reply]
`--enable-wayland-ime`

**Reply by purrion** [reply]
I'm using Sway + Fcitx5. 
I've tried adding `--gtk-version=4` and `--enable-wayland-ime` as per this [document](https://fcitx-im.org/wiki/Using_Fcitx_5_on_Wayland), but I can't use Fcitx5 in any Electron/Chromium applications (Brave, VSCode, Discord, Obsidian, Insomnia). They are currently running in Xwayland to work with the IME.
Has anyone faced this issue?

**Reply by Rizwan-Hasan** [reply]
For me `--enable-features=UseOzonePlatform` and `--ozone-platform=x11`worked.
